app/controllers/dashboard/repositories.js

```diff
@@ -99,6 +102,20 @@ export default Ember.Controller.extend({
     selectOrg(org) {
       let login = Ember.isBlank(org) ? undefined : org.get('login');
       return this.set('account', login);
+    },
+    star(repo) {
+      this.get('ajax').ajax(`/v3/repo/${repo.get('id')}/star`, 'POST')
+        .then(() => {
+          this.get('flashes')
+            .success(`You successfully starred ${repo.get('slug')}`);
+        });
+    },
+    unstar(repo) {
+      this.get('ajax').ajax(`/v3/repo/${repo.get('id')}/unstar`, 'POST')
+        .then(() => {
+          this.get('flashes')
+            .success(`You successfully unstarred ${repo.get('slug')}`);
+        });
     }
```

Comment:
I moved the starring action from the component to the controller. But I'm not sure if there is a better way of doing that with some api provided by Ember Data? 
Right now it doesn't trigger a refresh of the model, which makes sense - the model doesn't know about the POST request that way...
And everything else depends on that, but I guess there is a way to clean up the rest of the controller anyway